### PR TITLE
Certificates suggestions

### PR DESCRIPTION
**Pull request by gatemezing**
Added comments on Certificates.md  and legislations.md for review

### CODE REVIEW

File: ERADIS/CERTIFICATES.md
Comment (Interoperable-data) [inline ERADIS/CERTIFICATES.md:1]: Let us enrich example by having a beginning AFTER the issue date (this is the use case). To note is that issue data should not be after beginning date, but I think there are a few cases where this happened (SHACL sh:Warning).

**Review by Interoperable-data — CHANGES_REQUESTED** [review]

**Review by Certiman — CHANGES_REQUESTED** [review]
Line 3: check link markup

File: ERADIS/CERTIFICATES.md
Comment (Certiman) [reply ERADIS/CERTIFICATES.md:1]: Can we check these in the JENA KG?

Comment (Certiman) [reply ERADIS/CERTIFICATES.md:1]: Example updated

**Review by Certiman — COMMENTED** [review]
I have solved the http link differently

File: ERADIS/CERTIFICATES.md
Comment (Interoperable-data) [reply ERADIS/CERTIFICATES.md:1]: Update OK

**Review by Interoperable-data — APPROVED** [review]

**Review by Certiman — APPROVED** [review]
OK

**Pull request merged**